src/containerapp/azext_containerapp/_clients.py

```diff
@@ -31,15 +31,15 @@ def __init__(self):
         self.currTicker = 0
 
     def tick(self):
-        sys.stderr.write('\r')
-        sys.stderr.write(self.tickers[self.currTicker] + " Running ..")
-        sys.stderr.flush()
+        sys.stdout.write('\r')
```

Comment:
according to https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#general-patterns, only command output can go to stdout, and everything else should go to stderr (log/status/errors).